### ISSUE TITLE
Widgets are not the same than on admin page

### ISSUE THREAD

**Issue by olivierdalang** [ninapavlich/django-batch-uploader#4]
Hi,

Currently, the widgets of the batch uploader form default fields are not the same than on the regular admin page.
It's the case for at least :
- date fields, that display as plain text inputs, while there is a nice date picker in the normal admin
- foreign keys displayed as raw_id_fields (with or without grappelli's autocomplete), that display as normal foreign key fields (which is a blocker if there's a very long list of possible foreign keys, slowing down the page to death).

Note that this happens using grappelli, but from the screenshots I see you use it too.

Thanks !

**Comment by ninapavlich** [comment]
I completely agree! I have to do a little more research about how to initialize these with grappelli widgets.
